Automattic/isolated-block-editor
# Collab: Update peer caret positions more frequently

## Proposed changes

Broadcast selection changes to connected peers every time the selection is updated.

This deviates from the behavior in AsBlocks, where the selection data is only sent when content is actually modified. In practice, I think this caused quite a few clashes with other users, because their caret positions aren't really updated in real time.

## Testing instructions

1. `yarn storybook`
2. Open http://localhost:6006/?path=/story/collaboration--default in two browser windows.
3. Type a sentence and move your caret to different positions.
4. Your peer caret in the other window should update in real time.
